doc/languages-frameworks/python.section.md

```diff
@@ -1514,11 +1514,9 @@ Note: There is a boolean value `lib.inNixShell` set to `true` if nix-shell is in
 
 ### Tools {#tools}
 
-Packages inside nixpkgs are written by hand. However many tools exist in
-community to help save time. No tool is preferred at the moment.
-
-- [nixpkgs-pytools](https://github.com/nix-community/nixpkgs-pytools)
-- [poetry2nix](https://github.com/nix-community/poetry2nix)
+Packages inside nixpkgs must use the `buildPythonPackage` or `buildPythonApplication` function.
```

Comment:
```suggestion
Packages inside nixpkgs must use the `buildPythonPackage` or `buildPythonApplication` function
, because we can only provide security support for non-vendored dependencies.
```